src/styles/scss/style.scss

```diff
@@ -206,10 +191,10 @@ p {
 
 input {
   padding: 10px 20px;
-  border: 2px solid colors.$dandelion;
+  border: 2px solid #a9934c;
```

Comment:
I teraz sytuacja jest taka, że kilka kolorków bierzesz z libki, ale inne wstawiasz z palca. Masz więc dwa zestawy kolorów, gdy powinieneś mieć jeden, najlepiej w formie zmiennych SCSSowych.

Reply:
jak napisalem wyzej, SCSS zajmę się w osobnej PR i tam to zrobię